source/clear-linux/tutorials/multi-boot/multi-boot-rhel.rst

```diff
@@ -0,0 +1,168 @@
+
+         Figure 5: Red Hat: Configure swap partition
+
+#. Share the EFI partition that was created by |CL|. See Figure 6.
+
+   #. Expand :guilabel:`Unknown.`
+
+   #. Select :guilabel:`EFI System Partition / sda3`.
+
+   #. Under :guilabel:`Mount Point`, enter `/boot/efi`.
+
+   #. Click :guilabel:`Update Settings`.
+
+      .. figure:: figures/multi-boot-rhel-6.png
+
+         Figure 6: Red Hat: Configure EFI partition
+
+#. Click :guilabel:`Done`.
+
+#. Follow the remaining prompts to complete the installation of Red Hat.
+
+#. At this point, the ability to boot |CL| is lost because `Grub`
+   was set as the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add Red Hat as a boot option:
+
+   #. Boot into Red Hat.
+
+   #. Log in.
+
+   #. Get root privilege with the following command:
+
+      .. code-block:: console
+
+         $ sudo -s
+
+   #. Locate Fedora’s :file:`grub.cfg` file at the
+      :file:`/boot/efi/EFI/redhat/` directory and look for the primary Red
+      Hat :guilabel:`menuentry` section. The highlighted lines identify
+      the kernel and `initrd` filenames, root partition UUID, and
+      additional parameters used. This information is used to create a
+      new Systemd-Boot entry for Red Hat. See Figure 7.
+
+      .. figure:: figures/multi-boot-rhel-7.png
+
+         Figure 7: Red Hat: grub.cfg
+
+   #. Copy the kernel and `initrd` to the EFI partition.
+
+      .. code-block:: console
+
+         # cp /boot/vmlinuz-3.10.0-663.el7.x86_64 /boot/efi
+
+         # cp /boot/initramfs-3.10.0-663.el7.x86_64.img /boot/efi
+
+   #. Create a boot entry for Red Hat. The file must, at a minimum, contain
+      these settings:
+
+      +---------+---------------------------------------------------+
+      | Setting | Description                                       |
+      +=========+===================================================+
+      | title   | Text to show in the boot menu                     |
+      +---------+---------------------------------------------------+
+      | linux   | Linux kernel image                                |
+      +---------+---------------------------------------------------+
+      | initrd  | initramfs image                                   |
+      +---------+---------------------------------------------------+
+      | options | Options to pass to the EFI program or kernel boot |
+      |         | parameters                                        |
+      +---------+---------------------------------------------------+
+
+      See the `systemd boot loader documentation`_ for additional
+      details.
+
+      The *options* parameters must specify the root partition UUID and any
+      additional parameters that Red Hat requires.
+
+      .. note:: The root partition UUID used below is unique to this example.
+
+         .. code-block:: console
+
+            # cd /boot/efi/loader/entries
+
+            # vi redhat.conf
```

Comment:
Change "# vi redhat.conf" to "$ vi redhat.conf"

Reply:
Text edits deferred to next phase of updates.